tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -374,6 +375,11 @@ func (az *AzureProvider) createNIC(ctx context.Context, vnetName, subnetName, ns
 						Subnet:                    &subnet,
 						PrivateIPAllocationMethod: network.Dynamic,
 						PublicIPAddress:           &ip,
+						LoadBalancerBackendAddressPools: &[]network.BackendAddressPool{
```

Comment:
Please mention why you need to place it behind the LB here and in the commit and PR message.